website/documentation/getting-started/shoots/_index.md

```diff
@@ -0,0 +1,126 @@
+1. Deploy control plane incl. DNS records.
+1. Create nodes (VMs) and bootstrap kubelets.
+1. Deploy kube-system components to nodes.
+
+## How to Access a Shoot
+
+![](./images/access-shoot.png)
+
+Static credentials for shoots are discontinued with Kubernetes v1.27 in Gardener. Use short lived credentials instead. You can create/request tokens directly via Gardener or delegate authentication to an identity provider.
+
+A short-lived admin kubeconfig can be requested by using kubectl. If this is something you do frequently, consider switching to gardentctl (v2), which helps with it.
+
+![](./images/access-shoot-2.png)
+
+An alternative is to use an identity provider and issue oidc tokens.
+
+## What can you configure?
+
+With the basic configuration options known, it is time to discuss more possibilities. Gardener offers a variety of options to tweak the control plane's behavior - like defining an event TTL (default 1h), adding an OIDC configuration or activating some feature gate. You can alter the scheduling profile and define an audit logging policy. In addition, the control plane can be configured to run in HA mode (applied on a node or zone level), but note that it cannot be changed.
+
+In case you have specific requirements for the cluster internal DNS, Gardener offers a plugin mechanism for custom core DNS rules or optimization with node-local DNS. For more information, see [Custom DNS Configuration](https://github.com/gardener/gardener/blob/master/docs/usage/custom-dns-config.md) and [NodeLocalDNS Configuration](https://github.com/gardener/gardener/blob/master/docs/usage/node-local-dns.md).
+
+Another category of configuration options is dedicated to the nodes and the infrastructure they are running within. Every provider has their own perks and some of them are exposed. Check the detailed documentation of the relevant infrastructure provider extension.
+
+You can fine-tune the cluster-autoscaler or help the kubelet to cope better with your workload.
+
+## Worker Pools
+
+There are a couple of ways to configure a worker pool. One of them is to set everything in the Gardener dashboard. However, only a subset of options is presented there.
+
+![](./images/worker-pools-1.png)
+
+A slightly more complex way is to set the configuration through the yaml file itself.
+
+![](./images/worker-pools-2.png)
+
+You can configure much more, for example, the health timeout after which a machine is getting replaced. For more options, see the [Worker](https://github.com/gardener/gardener/blob/master/docs/api-reference/core.md#worker) API reference.
+
+## How to Change Things
+
+Since a shoot is just another Kubernetes resource, changes can be applied via kubectl. For convenience, the basic settings are configurable via the dashboard's UI. It also has a "yaml" tab where you can alter all of the shoot's specification. Once applied, the cluster will reconcile eventually and your changes become active (or cause an error).
```

Comment:
```suggestion
Since a Shoot is just another Kubernetes resource, changes can be applied via kubectl. For convenience, the basic settings are configurable via the dashboard's UI. It also has a "yaml" tab where you can alter all of the Shoot's specification in your browser. Once applied, the cluster will reconcile eventually and your changes become active (or cause an error).
```